Nextcloud App crash on clicking .vcf

### Steps to reproduce
1. Create a Backup of your (smartphone) contacts via App
2. Open the hidden .contact folder and click on the created .vcf
3. App crashes

### Expected behaviour
- Tell us what should happen
Open the file, download it or something else but no crash

### Actual behaviour
- Tell us what happens
The Nextcloud App crash

### Environment data
Android version: 9.0

Device model: Samsung Galaxy S8

Stock or customized system: Stock

Nextcloud app version: 3.12.0

Nextcloud server version: 19.0

### Logs
************ CAUSE OF ERROR ************

java.lang.RuntimeException: Unable to start activity ComponentInfo{com.nextcloud.client/com.owncloud.android.ui.activity.ContactsPreferenceActivity}: java.lang.ClassCastException: com.nextcloud.client.account.RegisteredUser cannot be cast to org.parceler.ParcelWrapper
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3114)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3257)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1948)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7050)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
Caused by: java.lang.ClassCastException: com.nextcloud.client.account.RegisteredUser cannot be cast to org.parceler.ParcelWrapper
	at org.parceler.Parcels.unwrap(Parcels.java:91)
	at com.owncloud.android.ui.activity.ContactsPreferenceActivity.onCreate(ContactsPreferenceActivity.java:96)
	at android.app.Activity.performCreate(Activity.java:7327)
	at android.app.Activity.performCreate(Activity.java:7318)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3094)
	... 11 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30120090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: samsung
Device: dreamlte
Model: SM-G950F
Id: PPR1.180610.011
Product: dreamltexx

************ FIRMWARE ************
SDK: 28
Release: 9
Incremental: G950FXXS9DTEA

--- a/src/main/java/com/owncloud/android/ui/activity/ContactsPreferenceActivity.java
+++ b/src/main/java/com/owncloud/android/ui/activity/ContactsPreferenceActivity.java
@@ -93,7 +93,7 @@ public class ContactsPreferenceActivity extends FileActivity implements FileFrag
                 transaction.add(R.id.frame_container, fragment);
             } else {
                 OCFile file = Parcels.unwrap(intent.getParcelableExtra(ContactListFragment.FILE_NAME));
-                User user = Parcels.unwrap(intent.getParcelableExtra(ContactListFragment.USER));
+                User user = intent.getParcelableExtra(ContactListFragment.USER);
                 ContactListFragment contactListFragment = ContactListFragment.newInstance(file, user);
                 transaction.add(R.id.frame_container, contactListFragment);
             }

